test/conftest.py

```diff
@@ -0,0 +1,32 @@
+import os
+
+import pytest
+
+from ThreeDiToolbox.datasource.netcdf_groundwater import NetcdfGroundwaterDataSource
+from ThreeDiToolbox.utils.patched_threedigrid import (
+    GridH5ResultAdmin,
+    GridH5AggregateResultAdmin,
+)
+
+
+cur_dir = os.path.dirname(__file__)
+data_dir = os.path.join(cur_dir, "data")
+bergermeer_dir = os.path.join(data_dir, "testmodel", "v2_bergermeer")
+
+gridadmin_path = os.path.join(bergermeer_dir, "gridadmin.h5")
```

Comment:
Netjes om een onderscheid te maken tussen `_dir` en `_path`.
Zorg er inderdaad voor dat je dat soort dingen consequent doet, dat scheelt veel kleine foutjes.

Idem `_file` (voor geopende files) en `_filename` (voor bestandsnamen), dat soort zaken.
Maakt niet veel uit *wat* je doet, als het maar consistent is :-)